docs/prerequisites/_index.md

```diff
@@ -1,31 +1,209 @@
+
+> 💡 The requirements above will also be dependant on the stateful applications that will be running in your cluster, therefore, it is recommended to review your stateful applications resources requirements when factoring in requirements for Ondat.
+
+### Architecture
+
+| **Name**                                                  | **Supported**    |
+| --------------------------------------------------------- | ---------------- |
+| [`x86-64 (64)`](https://en.wikipedia.org/wiki/X86-64) bit | `Yes`            |
+| [`arm64`](https://en.wikipedia.org/wiki/AArch64) bit      | `In Development` |
+
+### Linux Kernel
+
+- Below is the list supported Linux kernel versions and the maximum number of active volumes that can run per node in an Ondat cluster:
+
+| **Kernel Version** | **Maximum Number Of Mounted Volumes Per Node Allowed** |
+| ------------------ | ------------------------------------------------------ |
+| `v3.x.x`           | `256`                                                  |
+| `v4.x.x`           | `4096`                                                 |
+| `v5.x.x`           | `4096`                                                 |
+| `v6.x.x`           | `4096`                                                 |
+
+### Linux Asynchronous I/O (AIO)
+
+- When the Ondat Data plane component is operational, Ondat leverages [Linux Asynchronous I/O (AIO)](https://developer.ibm.com/articles/l-async/) contexts to initiate a number of I/O requests without having to block or wait for any to complete to process the next task - thus allowing you to boost performance for applications that are able to simultaneously overlap processing and I/O.
+- The Linux kernel uses the [`/proc/sys/fs/aio-max-nr`](https://www.kernel.org/doc/Documentation/sysctl/fs.txt) virtual file to set the value of the maximum number of *allowable* AIO concurrent requests. The `/proc/sys/fs/aio-nr` virtual file also provides information on the current number of asynchronous requests on the node.
+- Ondat requires `4` AIO contexts per deployed volume, whether it is a master or replica volume.
+  - Trying to provision additional volumes once the `aio-nr` value reaches the `aio-max-nr`, will cause a `io_setup` system call to fail and return a [`EAGAIN` error](https://www.kernel.org/doc/Documentation/sysctl/fs.txt), meaning that the resource becomes temporarily unavailable.
+- Therefore, if your nodes `aio-max-nr` kernel parameter value is less than `1048576`, a strong recommendation will be to set a new value in your `/etc/sysctl.conf` or through a custom configuration file in `/etc/sysctl.d/` to increase the number of AIO contexts that can be requested:
+
+| Kernel Parameter | Value     |
+| ---------------- | --------- |
+| `fs.aio-max-nr`  | `1048576` |
+
+> 💡 If your node(s) `aio-max-nr` kernel parameter value is less than `1048576`, the affected Ondat daemonset pods will report a warning error message in their logs.
```

Comment:
Cannot find this error message in a cluster up at the moment with fs.aio-max-nr = 65536. We should verify this

Reply:
I have removed this warning.